firedrake/output.py

```diff
@@ -357,7 +357,10 @@ def _prepare_output(self, function, cg):
         if cg:
             family = "Lagrange"
         else:
-            family = "Discontinuous Lagrange"
+            if function.function_space().mesh().ufl_cell() in (ufl.quadrilateral,):
+                family = "DQ"
+            else:
+                family = "Discontinuous Lagrange"
```

Comment:
It turns out the right thing to do is use "DG" if the cell is a simplex, otherwise "DQ" (UFL handles that case correctly). Can you update accordingly? You can kill the output.py one, because I think I fixed that when merging #1499.

Reply:
Great, thanks for the pointer. Done.